src/Discovery/Discovery.tsx

```diff
@@ -319,6 +321,9 @@ const Discovery: React.FunctionComponent<Props> = (props: Props) => {
         const formattedFields = formatSearchIndex(field);
         search.addIndex(formattedFields);
       });
+      if (config.studyPreviewField && searchableFields.includes('studyPreviewField.field')) {
```

Comment:
I'm not sure why these changes are in this file. The `studyPreviewFieldArr` was already going to be added into search on https://github.com/uc-cdis/data-portal/pull/1712/commits/22718225593b2a23807fde1de428994eec098e83#diff-22156db7794691aa0cc7b7966e41924e7f845742f177afd82d81f1fde56cdd93L332